_posts/en/newsletters/2025-12-19-newsletter.md

```diff
@@ -0,0 +1,1292 @@
+
+  Jesse Posner [highlighted existing research][news364 quantum primatives] that
+  indicates existing Bitcoin primitives like HD wallets, [silent
+  payments][topic silent payments], key aggregation, and [threshold
+  signatures][topic threshold signature] could be compatible with some of the
+  commonly referenced quantum-resistant signature algorithms.
+
+  Augustin Cruz [proposed][news qr cruz] a BIP to destroy coins that are
+  quantum-vulnerable with certainty. Subsequently, Jameson Lopp [started a
+  discussion][news qr lopp1] of how quantum-vulnerable coins should be handled,
+  which led to several ideas ranging from letting the quantum adversary have
+  them to destroying them. Lopp later [proposed][news qr lopp2] a concrete
+  [sequence of soft forks][BIPs #1895] that Bitcoin could implement, beginning
+  long before a CRQC is developed to gradually mitigate the threat of a quantum
+  adversary suddenly gaining access to many coins, while allowing holders time
+  to secure their coins.
+
+  Two proposals ([1][news qr sha], [2][news qr cr]) were made to enable
+  most existing coins to be secured in a way that could be recovered in the
+  event that Bitcoin disables quantum-vulnerable spends at some later point.
+  Briefly, the theorized sequence of events is 0) bitcoin holders ensure that
+  their current wallets have some hashed secret required for some spend path
+  1) a cryptographically relevant quantum computer (CRQC) is shown to be
+  eminent, 2) Bitcoin disables elliptic curve signatures, 3) Bitcoin enables a
+  quantum secure signature scheme, 4) Bitcoin enables one of these proposals
+  enabling prepared holders to claim their quantum-vulnerable coins. Depending
+  on the exact implementation, any address type (including P2TR with any
+  script path) could take advantage of these methods.
+
+  Developer Conduition demonstrated that [`OP_CAT`][BIP347] can be used to
+  implement Winternitz signatures, which provide a quantum-resistant
+  signature check at a cost of ~2000 vBytes per input. This is less costly
+  than the previously [proposed][rubin lamport] `OP_CAT`-based [Lamport
+  signatures][lamport].
+
+  Matt Corallo started a [discussion][news qr corallo] around the general idea
+  of adding a quantum-resistant signature-checking opcode to [tapscript][topic
+  tapscript]. Later, Abdelhamid Bakhta [proposed][abdel stark] native STARK
+  verification as one such opcode, and Conduition [wrote][conduition sphincs]
+  about their work optimizing SLH-DSA (SPHINCS) quantum resistant signatures as
+  another option. Any quantum resistant signature checking opcode including
+  `OP_CAT` added to tapscript could be combined with [BIP360][] to fully
+  quantum-harden Bitcoin outputs.
+
+  Tadge Dryja [proposed][news qr agg] one way in which Bitcoin could implement
+  general cross-input signature aggregation which would partially mitigate the
+  large size of post-quantum signatures.
+
```

Comment:
It seems to me that the paper out of Blockstream Research would fit very nicely here. Maybe something along the lines of:

```diff
+At the end of the year, Mikhail Kudinov and Jonas Nick [published][nick paper tweet] a 
+[paper][hash-based signature schemes] that provides an overview of hash-based signature
+schemes and discusses how those could could be adapted to suit Bitcoin’s needs.
+
+
+<!footnote>
+[nick paper tweet]: https://x.com/n1ckler/status/1998407064213704724
+[hash-based signature schemes]: https://eprint.iacr.org/2025/2203.pdf
```

Reply:
While we didnt cover this yet, I think its worth adding here. Adding.